### ISSUE TITLE
Look into content hosting options

### ISSUE THREAD

**Issue by daleglass** [overte-org/overte#632]
New users are having problems hosting things like avatars.

Dropbox has been tried, but is not an ideal option.

We could use researching the options out there, the difficulty of using them, whether they're friendly towards serving content to people, and whether there are any associated problems (pricing, etc). The results should then be placed prominently in the documentation.

**Comment by Zetaphor** [comment]
I was having this conversation with Lizard the other day, and we were investigating https://glitch.com

When you setup a site there its runtime is limited based on your subscription tier (free apps time out after 5 minutes, paid tier gets you 5 always-on apps), **however the file hosting is not affected by the container shutting down.**

I've verified this by creating a Glitch project and testing a direct link to a CSS file hours and even days after the last time it was accessed. Every time the file comes back immediately which tells me they're keeping static hosting around and only the container runtime is paused after the free-tier timeout. I just checked the URL again and it's been over a week since my last attempt. 

We could create a simple frontend that could also be hosted on Glitch that would allow a user to upload their files and receive a directory listing with direct links.

**Comment by vegaslon** [comment]
I also agree glitch seems like a solid option to host simple stuff. There also used to be keybase but the html frontend of that appears to have been removed after its latest corporate merger.

**Comment by daleglass** [comment]
Dropbox improved as of #634 and #636

**Comment by MarcusLlewellyn** [comment]
IPFS? *runs away*

**Comment by Zetaphor** [comment]
I've created a Glitch project that anyone can "remix" (copy) which provides a basic backend for uploading and managing files. Upon digging further I learned that Glitch made a change a while ago so now all static sites are always-on, this has the transient effect of any asset uploaded to glitch is always available even if the container application (node in this case) is not running.

Glitch limits the total amount of space you can have per project, but you can create an infinite number of copies of this project. I have no idea if using their service in this way is a violation of the ToS

https://overte-uploader.glitch.me/

**Comment by daleglass** [comment]
From Discord discussion: Discord plans to make content links temporary, so discord won't be usable anymore as of 2024

**Comment by ksuprynowicz** [comment]
From Discord discussion:
One of the difficulties of using Overte compared to other social VR applications is the need for hosting data on HTTP server. Since @daleglass is working on WebDAV support, NextCloud seems like a very good option that can be self-hosted and some companies even offer it for free. It turns out that it's possible to download files from there by HTTP requests:
https://stackoverflow.com/questions/75855784/download-a-file-from-a-nextcloud-public-link-pointing-to-a-folder-via-command
It's also possible to generate public links via API:
https://confluence.desy.de/pages/viewpage.action?pageId=202395716
With WebDAV and this we could have cloud storage similar to Neos/Resonite
An avatar share link could even be created just before joining the session and invalidated immediately after leaving the session.
